M2/Macaulay2/d/evaluate.d

```diff
@@ -1202,7 +1202,12 @@ parallelAssignmentFun(x:parallelAssignmentCode):Expr := (
     value := eval(x.rhs);
     when value is Error do return value else nothing;
     -- (x) = y should behave just like x = y
-    if nlhs == 1 then value = seq(value);
+    if nlhs == 1 then (
+	when value is a:Sequence do (
+	    -- unless y is a sequence of length 1, then it behaves like x = y#0
```

Comment:
Just to make sure, `(x) = (1,2,3)` should still be an error, correct?

Reply:
In the current draft, it's equivalent to `x = (1,2,3)`.  But for consistency, you're right -- it probably should be an error.  I'll push a new version here in a moment.

Reply:
@d-torrance @mahrud Is this agreed on, should I take a separate look?